src/js/include/render.js

```diff
@@ -0,0 +1,119 @@
+import Mustache from "mustache";
+import $ from "jquery";
+import compareVersions from "compare-versions";
+
+export function initResults(itemsToScan, activeOnly) {
+  const container = $("#wpe_pcc_results");
+  const template = $("#result-template").html().toString();
+  container.empty();
+
+  if (itemsToScan.plugins.length) {
+    container.append("<h3>Plugins</h3>");
```

Comment:
Wondering if we need to worry about translating this and the `Themes` string used below?

Reply:
Added translation support for JavaScript in https://github.com/10up/phpcompat/pull/3/commits/30409ddf3b143fa594a43525d06b4f126dc98188

The makepot workflow has changed (using wp-cli command instead of node-wp-i18n package, changed Makefile and described in the PR text)